src/main/java/com/gistpetition/api/petition/presentation/PetitionController.java

```diff
@@ -85,12 +83,21 @@
 
     @LoginRequired
     @PostMapping("/petitions/{petitionId}/agreements")
-    public ResponseEntity<Boolean> agreePetition(@PathVariable Long petitionId,
+    public ResponseEntity<Void> agreePetition(@RequestBody AgreementRequest agreementRequest,
+                                                 @PathVariable Long petitionId,
                                                  @LoginUser SimpleUser simpleUser) {
-        return ResponseEntity.ok().body(petitionService.agree(petitionId, simpleUser.getId()));
+        petitionService.agree(agreementRequest, petitionId, simpleUser.getId());
+        return ResponseEntity.ok().build();
     }
 
     @GetMapping("/petitions/{petitionId}/agreements")
+    public ResponseEntity<Page<AgreementResponse>> getPagefAgreements(@PageableDefault(sort = "createdAt", direction = Sort.Direction.DESC) Pageable pageable,
+                                                                      @PathVariable Long petitionId) {
```

Comment:
```suggestion
    public ResponseEntity<Page<AgreementResponse>> retrieveAgreements(@PathVariable Long petitionId,
                                                                      @PageableDefault(sort = "createdAt", direction = Sort.Direction.DESC) Pageable pageable) {
```
기존의 구현했던 방식과 통일성을 가져가면 좋겠다는 생각!

Pageable이 함수 시그니쳐에 들어나기 때문에, 명시적으로 메소드명으로 적어주지 않아도 될 것 같다는 생각이에요!
List를 메소드명에 명시하지 않는 개념과 비슷한 느낌입니다!